### ISSUE TITLE
hw02 ready for grading

### ISSUE THREAD

**Issue by rainerlempert** [rainerlempert/STAT545-hw-Lempert-Rainer#2]
@vincenzocoia @gvdr @ksedivyhaley @joeybernhardt @mynamedaike @pgonzaleze @derekcho

**Comment by aiod01** [comment]
Hello, I am Hyeongcheol Park. Thank you for your inviting me on your GitHub.

Overall, your GitHub homework is amazing. I was pretty impressive by how you organize things with proper skills we learned in the class.
 
 First of all, I was impressed by your decorating .md file. To be more specific, you used  library(knitr), kable() function, and prettyNum() function, which I didn’t know before. Even more, you wrote where others can get the information about your new functions. I appreciate you that I learned how to make my R results fancier!
 
 Secondly, your text was well made for readers. before analyzing was started,  you not only mentioned what are you going to do but also gave some questions to evoke readers interest. Because of your kindness, I didn’t have to pay a lot of attention to read the codes. 

Moreover, your exploring relationship between GDP and Population was well organized, because you pay attention to the good looking of the plots, by adding many options such as color, fill, labs, ggtitle, and so on. And x-scale logarithmic for clarity makes the plot easy to interpret. 

 Plus, you explained the reason why “filter(gapminder, country == c("Rwanda", "Afghanistan”))” is not proper with link, which was helpful for me too. 

If I should give you some suggestion for your homework, even though I actually do not find any flaw, I would say that it might be better to try other way to make a table for this part: kable(head(arrange(gapminder,desc(gdpPercap)))) this is just because the result on the screen was little bit hard to read, which was the only thing that is hard in your assignment. For one more thing, it would be nicer, I think, if you explain the reason that why the code“filter(gapminder, country == c("Rwanda", "Afghanistan”))”  search odd row for Rwanda, and even row for Afghanistan. 

To sum up, it was very nice work. your homework was made based on visitors to read, I found out. I enjoyed exploring your GitHub. Thank you and I hope you keep enjoy learning!

**Comment by ksedivyhaley** [comment]
**Smell test of data:**  Yes
**Explores at least one categorical/quantitative variable:** Yes
**Uses various ggplot types:** Yes (scatterplot, histogram, boxplot)
**Uses `filter()`, `select()`, and `%>%`:** Partial (no `select()`)
**Bonus (evaluate code, tables, more dplyr):** Yes, evaluate code, kable attempt, `mutate()`
**Reflection on process:** Yes

**Comments:**

- To fix your `kable` results, try setting the `format` argument.
- Neat attempt at skipping some countries in your table. Didn't quite work: you've got “NA etc ” in the middle instead of “etc etc”
- Nice find of `prettyNum()`
- Try piping into ggplot

Your mark will be distributed later. If you would like more feedback, please feel free to message me on slack.

**Comment by heathersummers** [comment]
Hi Rainer,

You did an amazing job with this assignment and it was very easy to navigate through your repository and assignment. The first part of the assignment ‘Explore the object’ was done very well and included answers with the appropriate R functions for all the required questions. For the second section of the assignment ‘Explore individual variables’ you went above and beyond and chose to explore two quantitative (pop and GDP per capita) and two qualitative (country and continent) variables. For the country variable, since all the countries had 12 observations I would also suggest looking at the spread of countries per continent (i.e. how many countries per continent) as that could be of interest when analyzing future plots (i.e. might be important to know there are only two countries in Oceania). The use of the function ‘summary()’ for exploring the quantitative variables was good for providing a number of descriptive statistics, however, I would suggest using a figure such as a histogram or barplot in order to visually represent the distribution of the data. For the third section of the assignment ‘Explore various plot types’ you explored several different types of plots including scatterplot, boxplot, and histogram and piped together functions such as ‘mutate()’ and ‘filter()’. The ggplot2 figures were easy to understand and showed quite a diverse use of functions to change the axis labels and titles, colours, and scales. Lastly, it was great to see you wanted to do more and were able to figure out what was incorrect with the code regarding the data for Rwanda and Afghanistan and I found it very helpful looking at the explanation on the stack overflow link you provided. Something else I found particularly helpful when I was reviewing your assignment was the link to the webpage that describes how to hide results, messages, and warnings that come up when you are loading packages in R Markdown. I will be using that code in my next assignment!! All the best with future assignments.

Heather
